lib/universal_peripherial.dart

```diff
@@ -3,43 +3,54 @@ import 'package:universal_chess_driver/peripherial.dart';
 import 'package:universal_chess_driver/peripherial_client.dart';
 import 'package:universal_chess_driver/utils.dart';
 
-class FeatureSupport {
-  Map<String, bool> _featureToEnabled = {
-    "msg": false,
-    "last_move": false,
-  };
-
-  bool get msg => _featureToEnabled["msg"]!;
-  set msg(bool value) {
-    _featureToEnabled["msg"] = value;
-  }
-
-  bool get lastMove => _featureToEnabled["last_move"]!;
-  set lastMove(bool value) {
-    _featureToEnabled["last_move"] = value;
-  }
-
-  void operator []=(String key, bool value) {
-    _featureToEnabled[key] = value;
-  }
-
-  Iterable<String> get keys => _featureToEnabled.keys;
+class UniversalPeripheralRound implements PeripherialRound {
+  String? get fen => null;
+  bool? get isSynchronized => null;
 }
 
 class UniversalPeripherial implements Peripherial {
   final PeripherialClient _client;
   final Central _central;
-  final FeatureSupport _features = new FeatureSupport();
   late _State _state;
+  List<String> _features = [];
+  List<String> _variants = [];
 
   UniversalPeripherial(this._client, this._central) {
```

Comment:
Consider `StringSerial` instead `PeripherialClient`:
```dart
abstract class StringSerial extends StringNotifier {
  Stream<String> stringStream;
  Future<void> send(String str);
}
```
Based on `StringNotifier` like `DataNotifier`as in: https://github.com/vovagorodok/ble_backends/blob/main/packages/ble_backend/lib/ble_serial.dart  
And implement:
```dart
class BleStringSerial extends StringSerial {
  BleStringSerial(BleSerial bleSerial) : _bleSerial(bleSerial) {}
  BleSerial _bleSerial;
  ...
}
```
It helps to use this protocol on  `UsbStringSerial`, `IrcStringSerial`,  etc..